tests/boards/nrf/coresight_stm/remote/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(nrf_coresight_stm)
+
+target_sources(app PRIVATE ../../../../../samples/boards/nordic/coresight_stm/src/main.c)
```

Comment:
Cannot `$ENV{ZEPHYR_BASE}` be used in here as in main application CMake, instead of `../`?

Reply:
My mistake.
At some point, I was switching from relative path to path based on $ZEPHYR_BASE.
I forgot to change in both CMakeLists.txt files.